README.md

```diff
@@ -283,6 +283,21 @@ To stop a container named `hnsd`, run:
 $ docker stop hnsd
 ```
 
+### OpenWRT
+
+To build hnsd as an OpenWRT package you'll need to rename `openwrt_Makefile` to `Makefile` 
+and put it to `your_openwrt_dir/package/net/hnsd` before building.
+Then you can use your `menuconfig` and select it.  
+Or you can use this command if you want to build on your SDK this package only:
+
+```bash
+$ make package/net/hnsd/compile V=s
+```
+
+Please keep in mind that `hnsd` needs `libunbound` and all of its dependencies
+such as `libsodium, libmnl, libevent2(all packs), libpthread, libnghttp2, python3-base,libprotobuf-c`
```

Comment:
Is this as easy as just
`opkg install libsodium libmnl,libevent2 libpthread libnghttp2 python3-base libprotobuf-c` ?

Reply:
Almost like it. I would suggest to try to enter them one by one and troubleshoot. For the most part of the supported architectures it should be easy (maybe for any, I just don't have a possibility to check them all). OpenWRT sometimes just not seeing that some packages are supported by default and you'll have to install them from the lowest dependencies to the top.